compute_endpoint/globus_compute_endpoint/MULTIUSER_SETUP.md

```diff
@@ -0,0 +1,334 @@
+   is that the software must be accessible to regular user accounts, so cannot
+   be installed in, say, `/root/`.)
+
+    ```console
+    # virtualenv /opt/globus-compute-endpoint
+    # . /opt/globus-compute-endpoint/bin/activate
+    # python -m pip install globus-compute-endpoint
+    ```
+
+1. Optionally, create an alias for easier typing in this session:
+
+    ```console
+    # alias globus-compute-endpoint=/opt/globus-compute-endpoint/bin/globus-compute-endpoint
+    ```
+
+1. Create the Multi-User Endpoint configuration with the `--multi-user` flag
+   (currently not advertised in `--help`) to the `configure` subcommand:
+
+    ```console
+    # globus-compute-endpoint configure --multi-user prod_gpu_large
+    Created multi-user profile for endpoint named <prod_gpu_large>
+
+        Configuration file: /root/.globus_compute/prod_gpu_large/config.yaml
+
+        Example identity mapping configuration: /root/.globus_compute/prod_gpu_large/example_identity_mapping_config.json
+
+        User endpoint configuration template: /root/.globus_compute/prod_gpu_large/user_config_template.yaml
+        User endpoint environment variables: /root/.globus_compute/prod_gpu_large/user_environment.yaml
+
+    Use the `start` subcommand to run it:
+
+        $ globus-compute-endpoint start prod_gpu_large
+    ```
+
+1. Setup the identity mapping configuration&nbsp;&mdash;&nbsp;this depends on
+   your site's specific requirements and may take some trial and error.  The
+   key point is to be able to take a Globus Auth Identity set, and map it to a
+   local username _on this resource_&nbsp;&mdash;&nbsp;this resulting username
+   will be passed to
+   [`getpwname(3)`](https://www.man7.org/linux/man-pages/man3/getpwnam.3.html)
+   to ascertain a UID for the user.  This file is linked in `config.yaml` (from
+   the previous step's output), and, per initial configuration, is set to
+   `example_identity_mapping_config.json`.  This file has a working (if
+   useless!) configuration; please refer to the [Globus Connect Server Identity
+   Mapping
+   Guide](https://docs.globus.org/globus-connect-server/v5.4/identity-mapping-guide/#mapping_recipes)
+   for help on updating this file, or reach out in [#help on the Globus Compute
+   Slack](https://funcx.slack.com/archives/C017637NZFA).
+
+1. Modify `user_config_template.yaml` as appropriate for the resources you want
+   to make available.  This file will be interpreted as a Jinja template, and
+   will become, after template processing with the user-provided variables, the
+   configuration that user endpoints will utilize.  The default configuration
+   (as created in step 4) has a basic working configuration, but uses the
+   `LocalProvider`.
+
+   Please look to [the documentation for a number of known working
+   examples](https://globus-compute.readthedocs.io/en/latest/endpoints.html#example-configurations)
+   (all written for single-user use) as a starting point.
+
```

Comment:
I think we should at least mention the `user_config_schema.json` here to inform admins that we support validation.